cmd/livepeer/livepeer.go

```diff
@@ -180,10 +180,12 @@ func main() {
 		}
 		lpmon.Instance().SetBootNode()
 	} else {
-		if err := n.Start(context.Background(), *bootID, *bootAddr); err != nil {
-			glog.Errorf("Cannot connect to bootstrap node: %v", err)
-			return
-		}
+		go func() {
+			if err := n.Start(context.Background(), *bootID, *bootAddr); err != nil {
```

Comment:
It looks like `n.Start` will only return an error if the node fails to set up the network protocol https://github.com/livepeer/go-livepeer/blob/master/core/livepeernode.go#L86 but will not return an error if it fails to connect to a bootnode https://github.com/livepeer/go-livepeer/blob/master/core/livepeernode.go#L95 so perhaps the error message should reflect that?

Reply:
`SetupProtocol()` can actually never return an error *right now* [1], which is the only reason I was somewhat comfortable going forward with this patch -- since it's not a regression over the current state of affairs. However, if the underlying behavior ever changes (eg, `SetupProtocol` returns an error, or we start handling errors from `Connect`) then we'd have to re-visit how that affects this bit of code. Given that, this patch is starting to look a bit too fragile for my tastes.

[1] https://github.com/livepeer/go-livepeer-basicnet/blob/4d00d3d14e576214b705c0b26790ee8b54feddd8/basic_network.go#L457-L474